tests/x11/kontact.pm

```diff
@@ -31,9 +31,10 @@ sub run {
         x11_start_program('echo -e "[General]\nfirst-start=false" >> ~/.kde4/share/config/kmail2rc', valid => 0);
     }
     x11_start_program('echo -e "[General]\nfirst-start=false" >> ~/.config/kmail2rc', valid => 0);
-
+    my $match_timeout //= 90;
+    $match_timeout = $match_timeout * 3 if (check_var('TEST', 'krypton-live'));
```

Comment:
I think it's safer to match on LIVE (or how it's called) rather than the test suite name